controllers/api/base.go

```diff
@@ -28,6 +28,6 @@ func (this *BaseController) ResponseWithError(errorMessage string, status int) {
 
 	err := controller.ServeJSON()
 	if err != nil {
-		log.Println("Failed to serve JSON response", err.Error())
+		logs.Info("Failed to serve JSON response", err.Error())
```

Comment:
Could we add a space before the error message? (Please check others change file also)

```suggestion
		logs.Info("Failed to serve JSON response: ", err.Error())
```

Reply:
✅  added in [fa66a9b](https://github.com/carryall/go-google-scraper-challenge/pull/54/commits/fa66a9b74c4a3b05e9ea1fa9fdda61c75327bd5b)